ClosedXML/Excel/Ranges/XLRangeAddress.cs

```diff
@@ -238,6 +242,9 @@ public String ToStringFixed(XLReferenceStyle referenceStyle)
 
         public String ToStringFixed(XLReferenceStyle referenceStyle, Boolean includeSheet)
         {
+            if (!IsValid)
```

Comment:
Should an invalid reference in this case not be `#REF!A1` instead of just `#REF!`?

Reply:
You are right.